### PR TITLE
link to new AutoSave content from existing event reference material

### PR DESCRIPTION
**Pull request by TristanD-MSFT**
Not sure if links are right (absolute vs. relative), but since we all contribute to making content great, just proposing an edit for folks to fix vs. starting an email thread to ask someone else to change. :)

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C14%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706142149507756-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C14%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706142150013903-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ ms.date: 06/08/2017
 
 Occurs after the workbook is saved.
```
Comment (JohnAustin-MSFT) [inline VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md]: Hi Tristan.  This is great new content... with a little quibble. We want to reference "internal links" - that is, content elsewhere in this repository - by using relative paths. The OPS topic: https://opsdocs.azurewebsites.net/en-us/opsdocs/partnerdocs/gfm?branch=master#markdown-extensions-supported-by-open-publishing describes the format of an internal link.  Look at the **Links** section of the topic.  There is additional guidance at: https://opsdocs.azurewebsites.net/en-us/opsdocs/partnerdocs/gfm?branch=master#resolve-file-path

File: VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ ms.date: 06/08/2017
 
 Occurs before any open workbook is saved.
 
+**NOTE:** In Office 365, Excel supports AutoSave, which enables the user's edits to be saved automatically and continuously. Following the guidance in [this article](https://msdn.microsoft.com/en-us/vba/office-shared-vba/articles/how-autosave-impacts-addins-and-macros) will ensure that running code in response to the **WorkbookBeforeSave** event will function as intended when AutoSave is enabled.
```
Comment (JohnAustin-MSFT) [inline VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md]: Same comment regarding links.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md)
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :white_check_mark:Succeeded | |

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md)
 - **Line 18**: **[Warning]** Invalid file link:(~/office-shared-vba/articles/how-autosave-impacts-addins-and-macros).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C21%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706212242163393-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookaftersave-event-excel.md)
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md)
[VBA/index.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/index.md) | :white_check_mark:Succeeded | |

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookaftersave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md)
 - **Line 17**: **[Warning]** Invalid file link:(~/office-shared-vba/articles/how-autosave-impacts-addins-and-macros).

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md)
 - **Line 18**: **[Warning]** Invalid file link:(~/office-shared-vba/articles/how-autosave-impacts-addins-and-macros).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C21%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706212251568759-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md (hunk 1 above)
Comment (JohnAustin-MSFT) [inline VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md]: Another tip:  GitHub URLs are case sensitive. This means that internal URLs such as "~/**o**ffice-**s**hared-**v**ba/articles/how-autosave-impacts-addins-and-macros" are not valid urls. The actual GitHub folder name is Office-Shared-VBA

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookaftersave-event-excel.md)
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md)
[VBA/index.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/index.md) | :white_check_mark:Succeeded | |

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookaftersave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md)
 - **Line 17**: **[Warning]** Invalid file link:(~/office-shared-vba/articles/how-autosave-impacts-addins-and-macros).

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md)
 - **Line 18**: **[Warning]** Invalid file link:(~/Office-Shared-Vba/articles/how-autosave-impacts-addins-and-macros).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706220004016577-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookaftersave-event-excel.md)
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md)
[VBA/index.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/index.md) | :white_check_mark:Succeeded | |

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookaftersave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md)
 - **Line 17**: **[Warning]** Invalid file link:(~/Office-Shared-VBA/articles/how-autosave-impacts-addins-and-macros).

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md)
 - **Line 18**: **[Warning]** Invalid file link:(~/Office-Shared-VBA/articles/how-autosave-impacts-addins-and-macros).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706220007413179-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/index.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706220040050994-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookaftersave-event-excel.md)
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md)
[VBA/index.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/index.md) | :white_check_mark:Succeeded | |

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookaftersave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md)
 - **Line 17**: **[Warning]** Invalid file link:(~/Office-Shared-VBA/articles/how-autosave-impacts-addins-and-macros).

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md)
 - **Line 18**: **[Warning]** Invalid file link:(~/Office-Shared-VBA/articles/how-autosave-impacts-addins-and-macros).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706220007312881-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :warning:Warning | | [Details](#user-content-VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md)
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/index.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/index.md) | :white_check_mark:Succeeded | |

<a name="VBA%2FExcel-VBA%2Farticles%2Fapplication-workbookbeforesave-event-excel.md"></a>
### [VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md)
 - **Line 18**: **[Warning]** Invalid file link:(~/Office-Shared-VBA/articles/how-autosave-impacts-addins-and-macros).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706220031085940-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/index.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706220032454129-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/Excel-VBA/articles/application-workbookbeforesave-event-excel.md) | :white_check_mark:Succeeded | |
[VBA/index.md](https://github.com/TristanD-MSFT/VBA-content/blob/patch-1/VBA/index.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C22%5Cf8b5b3f0-7390-2e9e-6625-a74fa1ae6f0a%5CPullRequest%5C201706220004523825-331%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**